terrawrap/models/wrapper_config.py

```diff
@@ -73,7 +73,8 @@ def __init__(
             plan_check: bool = True,
             envvars: Dict[str, AbstractEnvVarConfig] = None,
             backends: BackendsConfig = None,
-            depends_on: List[str] = None
+            depends_on: List[str] = None,
+            config: bool = True
```

Comment:
Adds config to wrapper config object. This allows us to filter directories that are tf config or not this is used in the create_wrapper_config_obj function in terrawrap/utils/config further down